webpage/src/installation/fedora.md

```diff
@@ -2,6 +2,13 @@
 
 There are QOwnNotes repositories for **Fedora 28 and higher**.
 
+::: tip
+QOwnNotes is provided in the Fedora repositories.
+That version is generally one or two patch versions behind the master repository available through the instrucitons below.
+For most users you can just issue ```dnf install qownnotes``` in a terminal window.  If you want the **most up-to-date version**, please continue reading.
```

Comment:
Please don't use multiline code blocks in a single line. Use inline code-blocks instead (one backtick, instead of 3). 😉

Reply:
OK, I will change that.  Sorry, I copied the formatting from another (multiline) area without considering it.